src/radio/index.js

```diff
@@ -149,6 +151,17 @@ class Radio extends EventEmitter {
   }
 
   addSong(link: string, user: Discord.User) {
+    const passedUrl = new URL(link);
+    if (passedUrl.pathname === '/playlist' && passedUrl.searchParams.has('list')) {
+      return playlistParser(String(passedUrl.searchParams.get('list')), this.app.config.youtube.key)
+        .then(items => items.map(item => this._addSong(item, user)))
+        .catch(reason => {});
```

Comment:
actually, let's make `addSong()`'s caller handle this
we'll want to do something different if web app or discord bot see this error